### ISSUE TITLE
[request] msquic/latest_stable

### ISSUE THREAD

**Issue by ZaMaZaN4iK** [conan-io/conan-center-index#1519]
### Package Details
  * Package Name/Version: **msquic/latest_stable**
  * Website: **https://github.com/microsoft/msquic**
  * Source code: **https://github.com/microsoft/msquic**

### Description Of The Library / Tool
Cross platform C implementation of the IETF QUIC protocol.

**Comment by SSE4** [comment]
we probably package some draft from here: 
https://github.com/microsoft/msquic/releases
`latest_stable` is a rolling release

**Comment by ZaMaZaN4iK** [comment]
@SSE4 Yeah, you are right. I meant that we should package something from releases, but they don't use semver :)

**Comment by nibanks** [comment]
MsQuic [does user semver](https://github.com/microsoft/msquic/blob/main/docs/API.md#versioning), and currently has three release branches (all backwards compatible, thus only the minor version change). Is there some reason why you think it doesn't use semver?
```
PS F:\msquic> git branch --all
...
  remotes/origin/release/1.0
  remotes/origin/release/1.1
  remotes/origin/release/1.2
```

**Comment by Jamaslab** [comment]
Is there any news about this library?
